packages/runtime-common/realm.ts

```diff
@@ -1704,10 +1704,25 @@ export class Realm {
     let useWorkInProgressIndex = Boolean(
       request.headers.get('X-Boxel-Use-WIP-Index'),
     );
-    let query = parseQueryString(new URL(request.url).search.slice(1));
+
+    let query: Query = parseQueryString(new URL(request.url).search.slice(1));
+
+    let htmlFormat = query.prerenderedHtmlFormat as string;
```

Comment:
I was actually thinking that htmlFormat was a separate query param. it seems weird to have it as part of the query since it's only for html queries specifically. I'd suggest not including it as part of the `Query` type since, unlike all the other fields in `Query` it is only for html queries.

Reply:
Yes, I had a dilemma about this. It was convenient to add it to the rest of the available query params, but on the  other hand, it's a very specific query that is used only for this endpoint. So i agree and will move it out to be a separate param.

Reply:
Addressed in https://github.com/cardstack/boxel/pull/1413/commits/6ebc4fa99162489d87db87ee7a50ec7948571880